README.md

```diff
@@ -69,6 +69,21 @@ In this example,
 
 The characters `"@"` and `"#"` identify the origin and the code, respectively. (We can always introduce additional prefix characters in the future if it turns out we need to include additional information in these messages.)
 
+Some sites use third-party `iframe`s for authentication. In such cases, the third-party `iframe`'s origin can be specified using a `"%"` field after the code.
```

Comment:
High level question: is the goal for this repo to specify specifically the sms format or to also go over the browser implementation that uses it.

That is, shouldn't this text go under the whatwg/html spec or the wicg/WebOTP spec?

Reply:
I think the format, and considerations for use by user agents. For instance, the spec talks about UA considerations that would apply to any feature in a browser that’d assist with providing codes to websites (e.g. Security Code AutoFill or WebOTP).

Reply:
See also my answer to this question here: https://github.com/WICG/sms-one-time-codes/issues/4#issuecomment-621436598